shopfloor/services/location_content_transfer.py

```diff
@@ -790,6 +790,30 @@ def stock_out_line(self, location_id, move_line_id):
         move_lines = self._find_transfer_move_lines(location)
         return self._response_for_start_single(move_lines.mapped("picking_id"))
 
+    def dismiss_package_level(self, location_id, package_level_id):
+        """Dismiss the package level.
+
+        The result package of the related move lines is unset, then the package
+        level itself is removed from the picking. This allows to move parts
+        of the package to different locations.
+
+        The user is then redirected to process the next line of the related picking.
+
+        Transitions:
+        * start_single: continue with the next line
+        """
+        location = self.env["stock.location"].browse(location_id)
+        if not location.exists():
+            return self._response_for_start(message=self.msg_store.record_not_found())
+        package_level = self.env["stock.package_level"].browse(package_level_id)
+        if not package_level.exists():
+            move_lines = self._find_transfer_move_lines(location)
+            return self._response_for_start_single(move_lines.mapped("picking_id"))
+        move_lines = package_level.move_line_ids
+        move_lines.write({"result_package_id": False})
```

Comment:
Maybe we could add a flag `shopfloor_do_first` and add the field in the move line sorter to ensure they are always returned first? So when we split the package, its lines are handled first

Reply:
We should use this: https://github.com/camptocamp/wms/pull/80